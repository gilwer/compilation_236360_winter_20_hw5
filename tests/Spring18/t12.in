//var, var to var
void main() {
	byte a = 5b;
	byte c = a;
	a = 6b;
	printi(a);
	
	printi(c);
	
	
	int d = 100;
	int e = d;
	d = 105;
	printi(d);
	
	printi(e);
	
	
	bool f = true;
	bool g = f;
	f = false;
	if (f) print("true"); else print("false");
	if (g) print("true"); else print("false");
	
	printi(a);
	printi(c);
	printi(d);
	printi(e);
	
}