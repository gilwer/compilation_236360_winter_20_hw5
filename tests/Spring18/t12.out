6
5
105
100
false
true
6
5
105
100
